airbyte-integrations/connectors/source-microsoft-teams/sample_files/sample_config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "tenant_id": "<your tenant ID>",
+  "client_id": "<your client ID>",
+  "client_secret": "<your client secret>"
+}
```

Comment:
In the `spec.json`, you indicated that you need 4 inputs and there is no `start_date `here.